roles/db_restore/tasks/restore_database.yml

```diff
@@ -28,6 +30,7 @@
 - name: restore database from backup
   postgresql_db:
     name: "{{ db }}"
+    login_host: "{{ postgres_login_host }}"
```

Comment:
why do we need this argument in this role? can't the playbook calling this role simply specify a different host depending on the type of infra (split or single server) is dealing with?

Reply:
I found if we supply the host, the hba authentication seems to go through TCP instead of direct peer-authentication of the system user, which messes it up in some cases. So if the host is remote, we supply it with `-h <host_ip>`, but if it's not we don't supply the `-h` flag at all, which is better.

Reply:
I've simplified a lot of this now